cmake/Linux.cmake

```diff
@@ -61,6 +61,12 @@ target_sources(pn_dev
   sample_app/main_linux.c
   )
 
+configure_file (
+  src/osal/linux/set_network_parameters
+  ${PROFINET_BINARY_DIR}/
+  COPYONLY
```

Comment:
Since replacing contents is unwanted, file(COPY) is maybe clearer and could be useful to specifically set permissions (executable bit).

Reply:
Done. The executable bit is set in the git repo, so I'm not modifying it during the copying.